roles/docker_storage_verify/tasks/main.yml

```diff
@@ -1,33 +1,19 @@
 ---
 # vim: set ft=ansible:
 #
-- name: Include distribution vars
-  include_vars: "{{ item }}"
-  with_first_found:
-    - "{{ ansible_distribution|lower }}-{{ ansible_distribution_version }}.yml"
-    - "{{ ansible_distribution|lower }}-{{ ansible_distribution_major_version }}.yml"
-    - "{{ ansible_distribution|lower }}.yml"
+- name: Set storage driver
+  set_fact:
+    storage_driver: 'overlay2'
 
-- name: Fail if required variables are not defined
-  when: driver is undefined
-  fail:
-    msg: "The required variables are not defined.  Check files under roles/docker_storage_verify/vars"
+- when: ansible_distribution == 'CentOSDev'
+  name: Set storage driver for CAHC
```

Comment:
I know we are trying to put the when statement first stylistically, but i think it makes more sense for the name to go first if there is a name, then the actual code underneath.

Reply:
no big deal, i can make that change